data/data/bootstrap/files/usr/local/bin/installer-masters-gather.sh

```diff
@@ -25,13 +25,23 @@ done
 
 echo "Gathering master journals ..."
 mkdir -p "${ARTIFACTS}/journals"
-for service in crio kubelet machine-config-daemon-host machine-config-daemon-firstboot openshift-azure-routes openshift-gcp-routes pivot sssd
+
+# Services [pcsd, pacemaker, corosync] are used in Two Node OpenShift with Fencing
+for service in crio kubelet machine-config-daemon-host machine-config-daemon-firstboot openshift-azure-routes openshift-gcp-routes pivot sssd pcsd pacemaker corosync
 do
     journalctl --boot --no-pager --output=short --unit="${service}" > "${ARTIFACTS}/journals/${service}.log"
 done
 
 journalctl -o with-unit --no-pager | gzip > "${ARTIFACTS}/journals/journal.log.gz"
 
+echo "Gathering pacemaker logs ..."
+mkdir -p "${ARTIFACTS}/pacemaker"
+cp /var/log/pcsd/pcsd.log "${ARTIFACTS}/pacemaker/pcsd.log"
+cp /var/log/pacemaker/pacemaker.log "${ARTIFACTS}/pacemaker/pacemaker.log"
+cp /var/log/cluster/corosync.log "${ARTIFACTS}/pacemaker/corosync.log"
```

Comment:
I know we discussed this elsewhere, so apologies for forgetting... these logs aren't expected to exist on all deployments, only TNF, so are we just depending on lack of `set -e` in the script to not fail when `cp` exits with an error, right?

Reply:
This is correct.

So the journald commands above will just write a file with no entries if given a service that doesn't exist.

The cp commands will throw an error like the following:
`cp: cannot stat '/var/log/pcsd/nothing.log': No such file or directory`

Because there's no `-e` the script completes without issue.

If it's preferred, I could add something like `|| true` to the end of these commands so that they would still exit 0 if the exiterr flag were set.

Reply:
Just asking for clarity.